src/components/Header.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react' // eslint-disable-line no-unused-vars
+import { SessionContext } from '../App'
+
+function Header () {
+  const { state } = React.useContext(SessionContext)
+
+  return (
+    <nav className='navbar sticky-top navbar-light bg-light'>
+      <div className='container-fluid'>
+        <a className='navbar-brand' href='#'>Pokemon Store</a>
+        <div className='d-flex'>
+          {state.cartCounter ? (
+            <span className='cart-items-counter badge badge-warning'>
```

Comment:
bad identation here

Reply:
Thanks for the comment.
Could you explain what's the reason and if it's not a problem could you also give an example of what would be the right way of indenting it?

Reply:
There is not a problem, just is not easy to read.
```javascript
<span
  id='cart-counter-badge'
  data-testid='cart-counter'
  className='cart-number-counter badge badge-pill badge-dark'>
  {state.cartCounter}
</span>
```